### DISCUSSION TITLE
NOT LOAD IMAGE !!!!!!!!! ELIKJUhasdlkjfhaqkl;sejb.kjb

### DISCUSSION THREAD

**Discussion by kaizer1** [pixijs/pixijs#7614] [Q&A] [answered]
function setup() {
  let sprite = new PIXI.Sprite(
    PIXI.loader.resources["src/assets/heartgrey.png"].texture
  );
  app.stage.addChild(sprite);
}
 
and console write error : 
 
(index):46 Uncaught TypeError: Cannot read property 'resources' of undefined
    at setup ((index):46)
    at (index):42 

 !!!!! 

And I checked the path and the presence of the picture a hundred times - I took the code example from your own training example - this is some kind of horror

**Comment by kaizer1** [comment]
thanks - I'll check it now (but it's strange, I copied from the example that I described on the site in the Tutorial section) - and I didn't have a load in another place (namely Add.Loader...)

**Comment by a-c-sreedhar-reddy** [comment]
https://pixijs.download/dev/docs/PIXI.Loader.html#resources This would be helpful I guess.

**Comment by kaizer1** [comment]
Sorry - but the same mistake 
<img width="348" alt="Снимок экрана 2021-07-19 110939" src="https://user-images.githubusercontent.com/5073933/126125836-b2db8a26-d4a7-43e5-aa68-730db5cd4c06.png">

**Comment by kaizer1** [comment]
My questioIs it even possible ? And if possible - is there a resource, or a tutorial, examples from which work, and are not just part of whose madness (not related in any way to Pixie)n very easy and simple - how to load image to pixi engine ( )

**Comment by a-c-sreedhar-reddy** [comment, marked as answer]
Can you try `app.loader.resources`
https://www.pixiplayground.com/#/edit/DvM66QEcKK12GP0prgJbm

**Comment by Ayfri** [comment]
Can you show your files list

**Comment by miltoncandelero** [comment]
> My questioIs it even possible ? And if possible - is there a resource, or a tutorial, examples from which work, and are not just part of whose madness (not related in any way to Pixie)n very easy and simple - how to load image to pixi engine ( )

It is possible.

The simplest example is what playground starts with:  
https://www.pixiplayground.com/

As for guides, if you want a more neutral guide: 
https://pixijs.io/guides/index.html

If you want a more opinionated guide: 
https://www.pixijselementals.com/#introduction

**Comment by ivanpopelyshev** [comment]
> PIXI.loader.resources

PIXI doesnt have `loader` constant in latest version.

> PIXI.Loader.resources

Loader is a class , not an instance. Either you use `new PIXI.Loader();` , either already created for you in application: "app.loader".

Docs are different for different versions of pixi. You also can take hints from your IDE, i hope you aren't doing it in notepad++?

**Comment by kaizer1** [comment]
no i use  Sublime Text 3

**Comment by miltoncandelero** [comment]
`Loader.shared` is also an alternative, however, you should be crashing earlier if you can't find your loader instance.

I mean, how are you doing the loading part?

```js
PIXI.Loader.shared.add('thingy', 'your/path/to/image.png')
    .load(startup);

function startup()
{
// ... your code here...
}
```

**Comment by kaizer1** [comment]
Thanks, I solved my problem with a hint @a-c-sreedhar-reddy - But other options are also interesting to me
